### DISCUSSION TITLE
Active Power Control Mode: : Zero Export Limitation

### DISCUSSION THREAD

**Discussion by crypto-kid** [wlcrs/huawei_solar#360] [General]
Hi every one, 

So in Denmark we often get 0 øre energy price and this results in it actually costing us money to send power to the grid. therefore I really need to be able to activate in the dongle configuration "Active Power Control Mode: : Zero Export Limitation" Is this already possible or at least is this an area the dev team can look at. I suspect as time goes on this will be come more important. Thanks for all the great work!!

**Comment by ghost** [comment]
https://github.com/wlcrs/huawei_solar/discussions/382#discussioncomment-6005823
